assets/components/thankYouIntroduction/thankYouIntroduction.scss

```diff
@@ -24,33 +24,36 @@
     }
   }
 
-  .svg-thank-you-hero-desktop {
+  .svg-circles-hero-desktop {
     position: absolute;
     top: 0;
     min-height: 100%;
     display: none;
 
-    @include mq($from: phablet) {
+    @include mq($from: desktop) {
       display: inline;
-      margin-left: -350px;
     }
+  }
 
-    @include mq($from: desktop) {
-      margin-left: -120px;
-    }
+  .svg-circles-hero-mobile-landscape {
+    position: absolute;
```

Comment:
I agree, but that would be quite a large refactor because this is also used in the CirclesIntroduction component and the way it has been written splits the different breakpoints out into different components, so we'd need to change the structure of that as well. 
I'd prefer to leave that out of this PR as it's big enough already!

Reply:
What happens if you use the `CirclesIntroduction` component?

Reply:
Fair enough, if @Ap0c's suggestion doesn't work please create a card to keep track of this refactor in the future.

Reply:
@Ap0c no because the text is arranged differently, in `CirclesIntroduction` the highlighted text is on the bottom, in this it's on the top. We probably could merge them with an extra parameter which says where you want the highlighting, but again it's non-trivial.

Reply:
Looking at @Amohkhan's designs on the card it appears that there is no highlight any more.